Source/SuperLinq/Move.cs

```diff
@@ -72,4 +72,30 @@ static IEnumerable<T> Core(IEnumerable<T> source, int bufferStartIndex, int buff
 				yield return e.Current;
 		}
 	}
-}
+
+	/// <summary>
```

Comment:
This is not your fault, just a consequence of parallel activities, but could you please update the xml docs to match the new style standard.

Reply:
Can deferred execution still be maintained if the 'from end' operator is used?

I have tried to avoid it by checking each index:
```cs
var length = 0;
if (range.Start.IsFromEnd || range.End.IsFromEnd || toIndex.IsFromEnd)
{
	length = source.Count();
}
var fromIndex = range.Start.IsFromEnd ? length - range.Start.Value : range.Start.Value;
var count = (range.End.IsFromEnd ? length - range.End.Value : range.End.Value) - fromIndex;
var to = toIndex.IsFromEnd ? length - toIndex.Value : toIndex.Value;
return source.Move
(
	fromIndex,
	count,
	to
);
```